dev-packages/e2e-tests/test-applications/solidstart-spa/src/routes/server-error.tsx

```diff
@@ -0,0 +1,17 @@
+import { withServerActionInstrumentation } from '@sentry/solidstart';
+import { createAsync } from '@solidjs/router';
+
+const getPrefecture = async () => {
+  'use server';
+  return await withServerActionInstrumentation('getPrefecture', () => {
+    throw new Error('Error thrown from Solid Start E2E test app server route');
+
+    return { prefecture: 'Kanagawa' };
+  });
+};
+
+export default function ServerErrorPage() {
```

Comment:
is this relevant then?

Reply:
Great catch, no it isn't!

Reply:
Did a double take and the `getPrefecture` function is extracted as a server function and then just executed as an api call on this run so I think it's fine to leave this in as is.